docs/docs/widgets/timeline.md

```diff
@@ -2,17 +2,18 @@
 id: timeline
 title: Timeline
 ---
+
 # Timeline
 
-The **Timeline** component can be used to do a visual representation of a sequence of events.
+The **Timeline** component provides a visual representation of a sequence of events. It's useful for displaying historical data, project milestones, or any chronological information in a clear and engaging format.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 ### Timeline data
```

Comment:
Please make sure all the headings are in title case only. Please refer the [style guide](https://docs.tooljet.com/docs/contributing-guide/documentation-guidelines/style-guide/#2-headings). Apply this to all headings in the document.